frontend/src/components/Label/Labels.tsx

```diff
@@ -8,6 +8,7 @@ const SHOW_MORE_TRESHOLD = 2;
 
 interface LabelsProps {
   expanded?: boolean;
+  header?: string;
```

Comment:
I don't think header is a correct name since we do not have header in Labels component... Maybe type or similar?